frontend/src/pages/Profile/index.tsx

```diff
@@ -60,8 +62,17 @@ const ProfilePage: React.FC = () => {
     );
   }
 
+  const notify = (message: string, isSuccess: boolean) => {
+    if (isSuccess) {
+      toast.success(message);
+    } else {
+      toast.error(message);
+    }
+  }
+
   return (
-    <AppMargin classname={classes.fullheight}>
+    userId &&
```

Comment:
oh whats this for haha what if u just define the update functions here and pass it to ur modals? you can render ur modals in this page i think

Reply:
oh cuz i wanted to pass userId to my modal but if i didnt add this they dont allow cuz it could be undefined. so i added this since if there is userprofile there will be userid.

Reply:
if i render here would it be a bit awkward? cuz the button is to open is in profile section

Reply:
ohh ok then!!

Reply:
ok i rendered in profile in the end 😅